aea/crypto/cosmos.py

```diff
@@ -236,7 +268,57 @@ def transfer(
         :param chain_id: the Chain ID of the Ethereum transaction. Default is 1 (i.e. mainnet).
         :return: tx digest if present, otherwise None
         """
-        raise NotImplementedError
+        result = self._try_get_account_number_and_sequence(crypto.address)
```

Comment:
Very adhoc, we can improve this in the weeks to come. But at least it works now and can be used.